README.md

```diff
@@ -1,47 +1,22 @@
 # VMZ: Model Zoo for Video Modeling
 
-VMZ is a Caffe2 codebase for video modeling developed by the Computer Vision team at Facebook AI. The aim of this codebase is to help other researchers and industry practitioners:
+VMZ is a Caffe2 and Pytorch codebase for video modeling developed by the Computer Vision team at Facebook AI. The aim of this codebase is to help other researchers and industry practitioners:
 + reproduce some of our research results and 
 + leverage our very strong pre-trained models. 
 
+
 Currently, this codebase supports the following models:
 + R(2+1)D, MCx models [[1]](https://research.fb.com/wp-content/uploads/2018/04/a-closer-look-at-spatiotemporal-convolutions-for-action-recognition.pdf).
-+ CSN models [[2]](https://arxiv.org/pdf/1904.02811.pdf).
++ CSN models [[2]](https://arxiv.org/pdf/1904.02811.pdf) (**note:pytorch implementation is buggy**).
```

Comment:
Bruno: does this mean it is slow due to missing the optimized 3D depthwise conv or it is actually buggy?

Reply:
It's slow due to missing optimized depthwise conv - also because of that it is less tested. So while I've been able to evaluate models successfully, I haven't done full optimized training run